src/importexport/musicxml/tests/data/testCueGraceNotes_ref.xml

```diff
@@ -691,7 +691,7 @@
         <duration>2</duration>
         <voice>1</voice>
         <type>quarter</type>
-        <accidental>natural</accidental>
+        <accidental cautionary="yes" parentheses="no">natural</accidental>
```

Comment:
Any idea anyone why accidentals on a note following a grace note on the same line/space are flagged as AccidentalRole::USER?